src/sentry/models/commitauthor.py

```diff
@@ -8,11 +10,29 @@
 
 class CommitAuthorManager(BaseManager):
     def get_or_create(self, organization_id, email, defaults, **kwargs):
+
+        # Truncate commit author name in case it is too long
+        author_name_source = None
+
+        if "name" in kwargs:
+            author_name_source = kwargs
+        elif "name" in defaults:
+            author_name_source = defaults
+
+        if author_name_source is not None and isinstance(
+            author_name_source["name"], six.string_types
+        ):
+            author_name_source["name"] = author_name_source["name"][:128]
```

Comment:
@dashed It would be great if you could create constants for the max length that we use in the model and use them directly here so the relationship is clear.